adapter/inventory.bzl

```diff
@@ -0,0 +1,27 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+def _inventory_gen(name, packages, out):
+  args = ""
+  for k, v in packages.items():
+    args += "-p %s:%s " % (k, v)
+  
+  native.genrule(
+      name = name+"_gen",
+      outs = [out],
```

Comment:
I would prefer -o options to make the tool standalone that takes input and produces output. However, I don't care for this particular case, since the stand alone tool that generates inventory has no use outside the mixer's build pipeline. Up to you.